### ISSUE TITLE
Shoutcast doesn't work

### ISSUE THREAD

**Issue by PedroRibeiro-zz** [segler-alex/rhythmbox-radio-browser#13]
Hi, i installed rhythmbox-radio-browser  on my ubuntu and shoutcast doesn't worek

**Comment by monkeystador** [comment]
I second that. Shoutcast recently changed their website. Guess the dataset used for the radiobrowser changed too.

**Comment by monkeystador** [comment]
Have to correct myself. There is a download button... it works.

**Comment by segler-alex** [comment]
hmm, shoutcast works for me. do you still have this issue?

**Comment by matrixcoder** [comment]
confirmed. Shoutcast (0) 
Ubuntu Lucid
Rythmbox 0.12.8
Installed from Repo

**Comment by kamar** [comment]
confirmed too
Ubuntu Lucid 64bit
Rhythmbox 0.12.8 from repositories

**Comment by jmmartin55** [comment]
Shoutcast doesn't work for me either - no genres, nothing.
Using 2.3.1 on Rhythmbox 0.12.8 on Mandriva 2010.1 64bit
Maybe the feed source is wrong?

**Comment by speedfreak** [comment]
Using 2.3.1-1, Rythmbox 0.13.1, Ubuntu Maverick 10.10

Shoutcast has been working, but now it doesnt. No genres, 0 stations.

**Comment by segler-alex** [comment]
not able to activate it again, because of new license. (this plugin can record, but i am not allowed to record). maybe deactivate recording just for shoutcast streams.
https://bugs.launchpad.net/ubuntu/+source/rhythmbox-radio-browser/+bug/691552

**Comment by harris4got** [comment]
Yes, deactivate record just for shoutcast streams. That would be a good solution, given the license restriction and by the way most of the people use it just for listening. And there is a cool new feature in the api. you can search for stations with 'now playing' info. Search for your fav artists and if you're lucky, you get a handful of stations. looking forward to the next version of radio browser.

**Comment by jmmartin55** [comment]
OK, I see that songbird has fixed their shoutcast plugin, and it now works like it used to.  Can we get our shoutcast module to work again, too?  Alex, what would you need to get things going again (besides more spare time to work on it, of course)?

**Comment by segler-alex** [comment]
Please look at:
https://bugs.launchpad.net/ubuntu/+source/rhythmbox-radio-browser/+bug/691552

**Comment by segler-alex** [comment]
if someone with an aol account is willing to give me an developer id, then i will add it again. i do not want to create an aol account, it was not neccessary before, and they want my data, which i don't want to give them :)
http://dev.aol.com/SHOUTcast/getting-started
